src/iogeneric/MimmoGeometry.hpp

```diff
@@ -134,6 +141,7 @@ enum WFORMAT{
  * - <B>Tolerance</B>:value of the geometric tolerance to be used;
  * - <B>Clean</B>: clean the geometry after read true 1/false 0;
  * - <B>FormatNAS</B>: 0-singlePrecision 1-doubleprecision for writing nas files;
+ * - <B>Parallel</B>: set if the read geometry is parallel true 1/false 0;
```

Comment:
@edoardolombardi may I suggest here to rename the Property as ParallelRestore just to remind the User that it works only for the restore of previously dumped patches?